e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -271,10 +268,10 @@ private void loadApiClientConfiguration(int apiPort) throws IOException, Interru
             }
         }
 
-        if(statusesBetween0And100.size() < 2) {
+//        if (statusesBetween0And100.size() < 2) {
             // Currently failing, add back when jobs take longer
             //Assert.fail("Did not receive more than 1 distinct progress values between 0 and 100");
-        }
+//        }
```

Comment:
Just remove the entire block.  If that test has not been re-enabled by now, not worth carrying forward.